build/versionize-docs.sh

```diff
@@ -68,8 +68,24 @@ for doc in "${mdfiles[@]}"; do
   fi
 done
 
-# Update API descriptions to match this version.
-$SED -ri -e "s|(releases.k8s.io)/[^/]+|\1/${NEW_VERSION}|" pkg/api/v[0-9]*/types.go
+# Update kubectl cmd files so that kubectl docs generated from them are as
+# expected.
+go_dirs=(pkg/kubectl/cmd)
+go_files=()
+for dir in "${go_dirs[@]}"; do
+  go_files+=($( find "${dir}" -name "*.go" -type f ))
+done
+# Update API descriptions as well
+go_files+=(pkg/api/v[0-9]*/types.go)
```

Comment:
Thanks for pointing that out.
Fixed. Am now running this script over the unversioned types.go as well